packages/react-sdk/README.md

```diff
@@ -106,7 +106,15 @@ function StartHuddleButton() {
     <>
       <button onClick={track}>Start huddle!</button>
       <button
-        onClick={() => requestFeedback({ title: "How do you like Huddles?" })}
+        onClick={(e) =>
+          requestFeedback({
+            title: "How do you like Huddles?",
+            position: {
+              type: "POPOVER",
+              anchor: e.target as HTMLElement,
```

Comment:
Let's use `e.currentTarget` instead as that allows you to position the widget relative to a parent og the triggered element. A bit more flexible. [See MDN](https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget)

```suggestion
        anchor: e.currentTarget as HTMLElement,
```